packages/protocol/contracts/common/UsingRegistryV2.sol

```diff
@@ -79,14 +79,6 @@ contract UsingRegistryV2 {
     _;
   }
 
-  /**
-   * @notice Returns the storage, major, minor, and patch version of the contract.
-   * @return The storage, major, minor, and patch version of the contract.
-   */
-  function getVersionNumber() external pure returns (uint256, uint256, uint256, uint256) {
```

Comment:
any reason not to remove this from `UsingRegistryV2BackwardsCompatible` as well?